src/natcap/invest/urban_nature_access.py

```diff
@@ -459,6 +517,147 @@ def execute(args):
     LOGGER.info('Finished Urban Nature Access Model')
 
 
+def _sum_rasters(greenspace_supply_list, target_sum_path):
```

Comment:
There are raster sum functions implemented in other models that you could import and reuse here, but I don't know if they're just as good. Given that these type of functions are duplicated a lot in invest, adding this one isn't a big deal. https://github.com/natcap/pygeoprocessing/issues/235

Reply:
Yep, I was previously using NDR's implementation and then switched (I don't remember why).  I just switched back, which seems fine.

But yes, I'd really rather see this addressed in a shared utility.